boards/arm/qemu_cortex_m3/doc/board.rst

```diff
@@ -6,9 +6,8 @@ ARM Cortex-M3 Emulation (QEMU)
 Overview
 ********
 
-The Zephyr kernel uses the qemu_cortex_m3 board configuration to emulate the TI
-LM3S6965 platform running on QEMU. It provides support for an ARM Cortex-M3 CPU
-and the following devices:
+This board configuration will use QEMU to emulate the TI LM3S6965 platform.
```

Comment:
When you write "This board configuration..." which board configuration are you referring to?  Why not mention it by name in this first sentence?

    The `BOARD=qemu_cortex_m3` board configuration ...